### ISSUE TITLE
Test Suite failing: Undefined method 'length' for nil:NilClass

### ISSUE THREAD

**Issue by buckethead1986** [learn-co-curriculum/tweet-shortener#10]
The test suites for selective_tweet_shortener and shortened_tweet_truncator are failing before it tests my code (which works as expected in the terminal, and fails even when the method is empty and has no reference to .length).

The error I get is 'Undefined method 'length' for nil:NilClass'.

**Comment by gj** [comment]
Hi there,

[Raising issues on GitHub](http://help.learn.co/improving-learn/contributing-to-lessons/raising-an-issue-with-a-lab-or-lesson) should be reserved for problems with the curriculum — not difficulties solving a lab, mistakes fixed via [a quick edit](http://help.learn.co/improving-learn/contributing-to-lessons/can-i-edit-a-lesson), or bugs with the Learn platform (such as [lights not turning green](http://help.learn.co/your-learn-account/learn-lights/lights-not-turning-green) or [IDE issues](http://help.learn.co/the-learn-ide)).

This issue would be perfect for the [Ask a Question](http://help.learn.co/ask-a-question/where-can-i-ask-a-question-about-a-lesson) (AAQ) feature, where the [Technical Coaches](http://help.learn.co/instructional-support/receiving-course-support/who-are-the-technical-coaches) (on [supported tracks](http://help.learn.co/instructional-support/receiving-course-support/does-my-program-or-course-have-technical-coach-support)) and fellow community members can help you figure it out. For some tips on how to take full advantage of AAQ, check out [this helpful article](http://help.learn.co/ask-a-question/how-to-optimize-time-in-ask-a-question) in the [Learn Help Center](http://help.learn.co/).

If, after consulting the Help Center and AAQ, you still feel that this is an issue with the curriculum itself, please reopen the issue and provide more detail about why it's a curriculum issue, the steps needed to reproduce the issue, and any additional relevant information about your development environment (local or IDE).

Thanks! 💙

**Comment by buckethead1986** [comment]
This was an issue with the curriculum, the test suite was not working
properly, and the lab was failing although my code was working properly in
the terminal.
I did use ask a question, but have received an answer once in about a
month. I have yet to have a technical coach get back to me through that
platform, and have raised several questions without responses.
I also reached out to my contact with the school, but was covering my bases
on github.

If an improper test suite is not a curriculum issue, what is? (So that I
can not raise false flags in the future)
